internal/ioutil/print.go

```diff
@@ -54,28 +54,28 @@ func PrintCertificateMap(w io.Writer, v []config.CertificateReference) error {
 	return tw.Flush()
 }
 
-func PrintVerificationResults(w io.Writer, v []*verification.SignatureVerificationOutcome, resultErr error) error {
+func PrintVerificationResults(w io.Writer, v []*verification.SignatureVerificationOutcome, resultErr error, digest string) error {
 	tw := newTabWriter(w)
 
-	overallResult := "success"
-	if resultErr != nil {
-		overallResult = "failure"
+	if resultErr == nil {
+		fmt.Fprintf(tw, "%s\n", digest)
+		return nil
 	}
-	fmt.Fprintf(tw, "OVERALL RESULT: %s\n", overallResult)
 
-	if resultErr != nil {
-		fmt.Fprintf(tw, "ERROR: %s\n", resultErr.Error())
-		printOutcomes(tw, v)
-	}
+	fmt.Fprintf(tw, "ERROR: %s\n\n", resultErr.Error())
+	printOutcomes(tw, v)
 
 	return tw.Flush()
 }
 
 func printOutcomes(tw *tabwriter.Writer, outcomes []*verification.SignatureVerificationOutcome) {
-	if len(outcomes) > 0 {
-		fmt.Fprintln(tw, "DETAILED ERRORS:")
-		for _, outcome := range outcomes {
-			fmt.Println(outcome.Error)
-		}
+	if len(outcomes) == 1 {
+		fmt.Println("1 signature failed verification, error is listed as below:")
+	} else {
+		fmt.Printf("%d signatures failed verification, errors are listed as below:\n", len(outcomes))
```

Comment:
If there are more than one failures, should we provide more info besides the errors? For example, when two outcome errors are different, we should print out which signature envelope leads to the corresponding outcome.Error.

Reply:
I discussed about that with Yi yesterday. It would look better if we can print out the signature digest for each error. But the result returned by the `verify` method does not include the digest. So we might need another PR to support that in `notation-go` verify method. We can make the change once @yizha1 completes the `Verify` spec.

Reply:
That makes sense. Otherwise, a single print out such as "signature is not produced by a trusted signer" does not really help the user to figure out what's going on. How about adding a TODO in the comment so we won't forget to add this part in the future?